style.css

```diff
@@ -0,0 +1,536 @@
+  width: 100%;
+  height: 60px;
+  color: var(--black-2);
+  font-size: 18px;
+}
+
+.btn-more i {
+  color: var(--red);
+  margin: 10px 15px;
+}
+
+.desktop-btn {
+  display: none;
+}
+
+.intro {
+  border: 5px solid #fff;
+  padding: 25px;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 30px;
+}
+
+.about-intro {
+  background-color: var(--white);
+  border: 3px solid rgb(223, 225, 229);
+  width: 85%;
+  text-align: center;
+  border-radius: 8px;
+  margin: 60px auto 10px auto;
+  line-height: 35px;
+}
+
+.about-contact {
+  padding: 5px;
+  font-size: 18px;
+  font-weight: 400;
+  margin-bottom: 0;
+}
+
+.past-event {
+  background-color: var(--white);
+  padding: 40px 0;
+  margin-top: 80px;
+}
+
+.past-event-intro {
+  padding: 0;
+  margin-top: -25px;
+  margin-bottom: 40px;
+}
+
+.past-event-img {
+  position: relative;
+  width: 80%;
+  height: 400px;
+  margin: 20px auto;
+}
+
+.past-event-img img {
+  width: 100%;
+  height: 100%;
+}
+
+.overlay {
+  position: absolute;
+  bottom: 0;
+  background: rgb(255 105 105 / 53%);
+  color: #f1f1f1;
+  width: 100%;
+  height: 100%;
+  margin: 0;
+}
+
+.text-on-img-1,
+.text-on-img-2 {
+  position: absolute;
+  top: 45%;
+  left: 40%;
+  color: var(--white);
+  font-size: 30px;
+  font-weight: 900;
+  margin: 0;
+}
+
+.text-on-img-2 {
+  top: 55%;
+  left: 18%;
+  font-weight: 400;
+  font-size: 25px;
+}
+
+.partner {
+  background: rgb(66, 65, 70);
+  height: 200px;
+  padding: 40px;
+}
+
+.partner-mob-about {
+  display: block;
+}
+
+.partner-desktop-home {
+  display: none;
+}
+
+.partner h2 {
+  margin: 0;
+  padding: 10px;
+  color: var(--grey);
+}
+
+.partner img {
+  padding: 0 40px;
+  width: 100px;
+  height: 100px;
+}
+
+.featured-speaker {
+  width: 100%;
+  background: var(--white);
+  padding: 40px 0;
+}
+
+.speaker-container {
+  margin: auto;
+  padding: 30px;
+  width: 75%;
+  display: grid;
+  grid-template-columns: auto;
+}
+
+.speaker {
+  display: flex;
+  grid-column-gap: 10px;
+  margin: 25px 10px;
+}
+
+.speaker-3,
+.speaker-4,
+.speaker-5,
+.speaker-6 {
+  display: none;
+}
+
+.speaker img {
+  display: inline-block;
+  width: 150px;
+  height: 150px;
+}
+
+.speaker div {
+  width: 60%;
+}
+
+.speaker-position {
+  color: var(--red);
+  font-style: italic;
+  font-size: 15px;
+}
+
+.grey-hr {
+  width: 10%;
+  text-align: left;
+  background-color: var(--grey);
+  border: none;
+  margin: 0;
+}
+
+.about {
+  padding-top: 60px;
+}
+
+.footer {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  text-align: left;
+  height: 100px;
+  padding: 40px;
+  background: var(--white);
+}
+
+.logo,
+.footer .footer-logo {
+  font-family: 'Dancing Script', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  letter-spacing: -3px;
+  text-decoration: none;
+  color: var(--black-2);
+}
+
+.footer p {
+  padding: 0 10px;
+  margin-top: 5px;
+}
+
+.desktop-home-footer {
+  display: none;
+}
+
+@media screen and (min-width: 769px) {
```

Comment:
- [ ] The mobile version of your design is overlapping, hence **not responsive**. Kindly endeavor to improve upon this aspect of your design.

**You mobile version**
![image](https://user-images.githubusercontent.com/12745474/132687306-6482dd69-40a9-494d-a2ad-d119d90995fb.png)

**The original mobile version**
![image](https://user-images.githubusercontent.com/12745474/132687958-69b8e9da-c14d-4cee-814d-e6538c7f941e.png)